boggs/src/main/java/com/hartwig/healthchecks/boggs/healthcheck/prestast/PrestatsExtractor.java

```diff
@@ -33,43 +32,29 @@
 
 	public PrestatsReport extractFromRunDirectory(String runDirectory) throws IOException, EmptyFileException {
 		List<PrestatsDataReport> summaryData = getSummaryFilesData(runDirectory);
-		List<PrestatsDataReport> fastqcData = getfastqFilesData(runDirectory);
+		PrestatsDataReport fastqcData = getfastqFilesData(runDirectory);
 
-		if (summaryData.isEmpty() || fastqcData.isEmpty()) {
+		if (summaryData.isEmpty() || fastqcData == null) {
 			throw new EmptyFileException(String.format(EMPTY_FILES_ERROR, runDirectory));
 		}
 
 		PrestatsReport prestatsData = new PrestatsReport(CheckType.PRESTATS);
 		prestatsData.addAllData(summaryData);
-		prestatsData.addAllData(fastqcData);
+		prestatsData.addData(fastqcData);
 
 		return prestatsData;
 	}
 
-	List<PrestatsDataReport> getfastqFilesData(String runDirectory) throws IOException {
-		List<Path> fastqcFiles = Files.walk(new File(runDirectory).toPath())
-				.filter(p -> p.getFileName().toString().contains(FASTQC_DATA_FILE_NAME)).sorted()
-				.collect(toCollection(ArrayList<Path>::new));
-
-		List<PrestatsDataReport> fastqcData = fastqcFiles.stream().map(path -> {
-			PrestatsDataReport prestatsDataReport = null;
-			try {
-				Map<String, String> data = getFastqcData(path);
-				String totalSequences = data.get(TOTAL_SEQUENCES);
-				if (totalSequences != null) {
-					String status = "PASS";
-					if (Long.parseLong(totalSequences) < MIN_TOTAL_SQ) {
-						status = "FAIL";
-					}
-					prestatsDataReport = new PrestatsDataReport(status, TOTAL_SEQUENCES, data.get(FILENAME));
-				}
-			} catch (IOException e) {
-				LOGGER.error(String.format("Error occurred when reading file. Will return empty stream. Error -> %s",
-						e.getMessage()));
-			}
-			return prestatsDataReport;
-		}).collect(Collectors.toList());
-		return fastqcData;
+	PrestatsDataReport getfastqFilesData(String runDirectory) throws IOException, EmptyFileException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/hmf/health-checks/file/2780521380/issues/source?bid=3363458&fileBranchId=3365368#l48)